shared/js/content-scripts/element-hiding.js

```diff
@@ -0,0 +1,87 @@
+var isTop = window === window.top;
+var contentScript = {};
+var csPort = chrome.runtime.connect({name:"cs-port"});
+var frameId;
+var mainFrameUrl;
+var possibleTargets = [];
+var foundScripts = [];
+var foundFrames = [];
+
+
+window.addEventListener("message", (e) => {
+    if (isTop) {
+        if (e.data.frameUrl) {
+            foundFrames = document.getElementsByTagName('iframe');
+            let i = foundFrames.length;
+            while (i--) {
+                let frame = foundFrames[i];
+                if (frame.src && !e.data.blockedRequests.includes(frame.src)) {
+                    frame.contentWindow.postMessage({frameId: frame.id, mainFrameUrl: document.location.href, blockedRequests: e.data.blockedRequests}, frame.src);
+                }
+            }
+        } else if (e.data.frameId) {
+            console.log("hiding top level iframe with id", e.data.frameId);
+            let blockedFrame = document.getElementById(e.data.frameId);
+            blockedFrame.hidden = true;
+            blockedFrame.style.setProperty('display', 'none', 'important');
+            if (blockedFrame.parentNode.children.length === 1) {
+                blockedFrame.parentNode.style.setProperty('display', 'none', 'important');
+            }
+        }
+    } else {
+        if (e.data.frameId) {
+            frameId = e.data.frameId;
+            mainFrameUrl = e.data.mainFrameUrl
+            contentScript.locateBlockedFrames(e.data.blockedRequests);
+        }
+    }
+});
+
+csPort.onMessage.addListener((m) => {
+    if (!isTop) {
```

Comment:
Can you switch this to `if (isTop) {} else {}` to stay consistent with the others.